PilnaKalade/Assets/Scripts/UI/BattleScene.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BattleScene : MonoBehaviour
+{
+    public void OpenSettingsMenu()
+    {
+        TransitionController.TransitionTo(2);
```

Comment:
what is 2? Could you create a constant which would describe what this 2 means?

Reply:
Added constants in https://github.com/Randwarf/KTU_PilnaKalade/pull/15